assets/models/system/databricks-dolly-v2-12b/spec.yaml

```diff
@@ -4,10 +4,12 @@ path: ./
 properties:
   SHA: 19308160448536e378e3db21a73a751579ee7fdd
   datasets: databricks/databricks-dolly-15k
+  finetuning-tasks: text-classification, token-classification
```

Comment:
🚀

Reply:
text-gen has issue. For which there is a bug on FT team: https://msdata.visualstudio.com/Vienna/_workitems/edit/2835559